api/apiServer.py

```diff
@@ -0,0 +1,111 @@
+import vagrant
+import os
+import boto3
+import json 
+from collections import defaultdict
+from fastapi import FastAPI
+
+app = FastAPI()
+
+@app.get("/")
+def root():
+    return {"message": "Welcome!"}
+    
+    #Provison VB API
+    
+@app.get("/provision/vb/createInstance")
+def root():
+    vagrantfile = "/path/to/vb/vagrantfile"
```

Comment:
maybe we can use default vagrantfile location for example “api/config/“ or “api/files”